.github/workflows/load-tester-dockerimage.yml

```diff
@@ -17,17 +17,22 @@ jobs:
       env:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
           DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-      run: docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD 
+      run: docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+
+    - name: Tags
+      id: tags
+      uses: livepeer/action-gh-release-tags@v0
+      with:
+        tags-prefix: load-tester
+        force-latest: true
+
     - name: Build the Docker image
-      # run: docker build . --file Dockerfile --tag livepeer/streamtester:$(date +%s)
-      run: docker build . --file Dockerfile.load-tester --tag livepeer/streamtester:load-tester --build-arg version=$(git describe --dirty)
-    - name: Get tag name
-      id: get_version
-      run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
-    - name: Tag version
-      if: startsWith(github.ref, 'refs/tags/v')
       run: |
-        docker tag livepeer/streamtester:load-tester livepeer/streamtester:load-tester${{ steps.get_version.outputs.VERSION }}
-        docker push livepeer/streamtester:load-tester${{ steps.get_version.outputs.VERSION }}
+        TAGS=(${{ steps.tags.outputs.tags }})
+        docker build . --file Dockerfile.load-tester ${TAGS[@]/#/--tag livepeer/streamtester:} --build-arg "version=${{ steps.tags.outputs.version }}"
```

Comment:
This... is not easy to Google. It's like a bash inline foreach...? Looks cool.

Reply:
It is! Got it from here: https://stackoverflow.com/questions/20366609/prefix-and-postfix-elements-of-a-bash-array/30971572

Maybe better to switch to the `sh`-compatible way of doing this, as I did it on `livepeer-com`? Something like `$(printf ' -t livepeer/streamtester:%s' $TAGS)`

Reply:
Done!